spaces/verified.json

```
@@ -115,6 +115,15 @@
   "astardegens.eth": 1,
   "spookyswap.eth": 1,
   "hbot-prp.eth": 1,
+  "beanstalkfarms.eth": 1,
+  "elyfi-bsc.eth": 1,
+  "kogecoin.eth": 1,
+  "futera.eth,
```

@metrox-eth you miss a quote here.
```suggestion
  "futera.eth",
```